pkg/cronjob/es_rollover.go

```diff
@@ -131,14 +133,20 @@ func lookback(jaeger *v1.Jaeger) batchv1beta1.CronJob {
 
 func esScriptEnvVars(opts v1.Options) []corev1.EnvVar {
 	var envs []corev1.EnvVar
-	if val, ok := opts.Map()["es.index-prefix"]; ok {
-		envs = append(envs, corev1.EnvVar{Name: "INDEX_PREFIX", Value: val})
-	}
-	if val, ok := opts.Map()["es.username"]; ok {
-		envs = append(envs, corev1.EnvVar{Name: "ES_USERNAME", Value: val})
-	}
-	if val, ok := opts.Map()["es.password"]; ok {
-		envs = append(envs, corev1.EnvVar{Name: "ES_PASSWORD", Value: val})
+	options := opts.Map()
+
+	for k, v := range map[string]string{
+		"es.index-prefix": "INDEX_PREFIX",
```

Comment:
Adding just a pointer to env vars in the script https://github.com/jaegertracing/jaeger/blob/master/plugin/storage/es/esCleaner.py#L20